cupy/__init__.py

```diff
@@ -1065,6 +1065,12 @@ def issctype(rep):
         return False
 
 
+if _numpy.__version__ >= "2":
+    from numpy import bool  # NOQA
+    from numpy import long  # NOQA
+    from numpy import ulong  # NOQA
```

Comment:
Our policy for v14 is to provide shims to keep consistent API for both NumPy v1/v2 environment. Could you add these for NumPy v1 case, and also add XXX comment as seen below?

```py
from numpy import bool_ as bool
from numpy import int_ as long
from numpy import uint as ulong
```